packages/lang-server/src/diagnostics.ts

```diff
@@ -48,8 +48,8 @@ export const getDiagnostics = async (
       .map(err => workspace.transformError(err))
       .map(async errPromise => {
         const err = await errPromise
-        return err.sourceFragments.map(f => {
-          const range = f.subRange || f.sourceRange
+        return err.sourceLocations.map(sl => {
+          const range = sl.subRange || sl.sourceRange
```

Comment:
not new code, but still
```suggestion
          const range = sl.subRange ?? sl.sourceRange
```